tests/src/rules/no-internal-modules.js

```diff
@@ -14,6 +14,13 @@ ruleTester.run('no-internal-modules', rule, {
       filename: testFilePath('./internal-modules/plugins/plugin.js'),
```

Comment:
I've added tests, but maybe not enough. Don't hesitate to ask me to implement other tests if you see something I've missed :)